telegram/ext/filters.py

```diff
@@ -104,13 +101,54 @@ def __repr__(self):
             self.name = self.__class__.__name__
         return self.name
 
+
+class MessageFilter(BaseFilter, ABC):
+    """Base class for all Message Filters. In contrast to :class:`UpdateFilter`, the object passed
+    to :meth:`filter` is ``update.effective_message``.
```

Comment:
Could you maybe add the "If you want to create your own filters create a class inheriting..." text to the MessageFilter and UpdateFilter docstrings too?
Or maybe just "See BaseFilter for details on how to create your own custom filters" or similar

Reply:
jup